ui/main/src/app/modules/card/components/card-body/card-body.component.ts

```diff
@@ -188,6 +195,16 @@ export class CardBodyComponent implements OnChanges, OnInit, OnDestroy {
         }
     }
 
+    private computeCardHasBeenRead() {
+        this.card = {...this.card,
+            hasBeenRead: OpfabStore.getLightCardStore().isLightCardHasBeenRead(this.card)
+        }
+    }
+
+    private computeCardHasBeenAcknowledged() {
+        this.card.hasBeenAcknowledged = OpfabStore.getLightCardStore().isLightCardHasBeenAcknowledged(this.card);
+    }
+
```

Comment:
the ack status is not used directly in card-body.component , i suggest to avoid dealing with ack here but do the processing where it is needed i.e in setAcknowledgeButtonVisibility  method in card-ack.component.ts